openbas-api/src/main/java/io/openbas/execution/ExecutionExecutorService.java

```diff
@@ -27,30 +29,83 @@ public class ExecutionExecutorService {
   public void launchExecutorContext(Inject inject) {
     // First, get the agents of this injects
     List<Agent> agents = this.injectService.getAgentsByInject(inject);
+    List<Agent> inactiveAgents = agents.stream().filter(agent -> !agent.isActive()).toList();
```

Comment:
nitpick: could you add comment explaining the use of the removeall call like we talked about this morning?